mod_reforged/msu_systems/mod_settings.nut

```diff
@@ -13,3 +13,10 @@ tacticalTooltipPage.addRangeSetting("CollapseActivesWhenX", 5, 0, 20, 1, "Collap
 tacticalTooltipPage.addBooleanSetting("TacticalTooltip_CollapseAsText", false, "Collapse as Text", "If enabled, then skills collapse using their names as text, otherwise they collapse using their icons.");
 tacticalTooltipPage.addBooleanSetting("ShowStatusPerkAndEffect", true, "Show Status Perk And Effect", "Some Perks are also Status Effects. Usually their Effect is hidden until some condition is fulfilled. When this setting is enabled, these perks show up in the Perks category even when they show up under Effects (e.g. when their effect is active). When disabled, when they appear under Effects, they will be hidden from the Perks category. This can help save space on the tooltip.");
 tacticalTooltipPage.addBooleanSetting("HeaderForEmptyCategories", false, "Show Header for empty categories");
+
+::Reforged.Mod.Keybinds.addSQKeybind("WaitRound", "ctrl+space", ::MSU.Key.State.Tactical, function()
```

Comment:
For convention let's prefix the scope of the keybind to its ID. Something like `tactical_WaitRound`. We do this in MSU too and it's good to keep things organized.

Reply:
I actually prefer 2 as well